test/remote/gateways/remote_global_collect_test.rb

```diff
@@ -12,6 +12,22 @@ def setup
     @cabal_card = credit_card('6271701225979642', brand: 'cabal')
     @declined_card = credit_card('5424180279791732')
     @preprod_card = credit_card('4111111111111111')
+    @apple_pay = network_tokenization_credit_card('4567350000427977',
```

Comment:
:question:
you're using the number '4567350000427977' for google_pay and apple_pay, is it by a special reason o  we can use also some of these [test data](https://docs.spreedly.com/reference/test-data/ ) ?

Reply:
Yes, sure, you could use the test card of [test data](https://docs.spreedly.com/reference/test-data/), the way to change between successful or unsuccessful transactions with apple/google pay for GlobalCollect is in the amount value.